File: easy/kotlin/onboarding.kt
fun main() {
    while (true) {
        val enemy1 = readLine()!!
        val dist1 = readLine()!!.toInt()
        val enemy2 = readLine()!!
        val dist2 = readLine()!!.toInt()

        if (dist1 < dist2) {
            println(enemy1)
        } else {
            println(enemy2)
        }
    }
}

File: easy/c#/onboarding.cs
using System;

class Program {
    static void Main() {
        while (true) {
            string enemy1 = Console.ReadLine();
            int dist1 = int.Parse(Console.ReadLine());
            string enemy2 = Console.ReadLine();
            int dist2 = int.Parse(Console.ReadLine());

            if (dist1 < dist2) {
                Console.WriteLine(enemy1);
            } else {
                Console.WriteLine(enemy2);
            }
        }
    }
}
